### PR TITLE
Refactor/tx details page

### PR DESCRIPTION
**Pull request by gesseekur**
https://app.asana.com/0/1140360345877287/1179140011942249

### CODE REVIEW

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue [hunk 1]
```diff
@@ -173,121 +143,103 @@ export default class PageDetailsTxs extends Vue {
...
-            const receipt = transaction.receipt!
-
+            console.error('tx', this.transaction)
+            // const receipt = transaction.receipt!
             details = [
                 {
                     title: this.$i18n.t('block.number'),
-                    detail: transaction.blockNumberFormatted,
-                    link: `/block/number/${transaction.blockNumber}`
+                    detail: this.transaction.blockNumber,
```
Comment (olgakup) [inline newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue:149]: formatNumber()

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue [hunk 2]
```diff
@@ -173,121 +143,103 @@ export default class PageDetailsTxs extends Vue {
...
                     title: this.$i18n.t('common.amount'),
-                    detail: `${transaction.valueFormatted.value} ${this.$i18n.t(`common.${transaction.valueFormatted.unit}`)}`,
-                    tooltip: transaction.valueFormatted.tooltipText ? `${transaction.valueFormatted.tooltipText} ${this.$i18n.t('common.eth')}` : undefined
-                },
-                this.toDetail(transaction),
-                {
-                    title: this.$i18n.tc('tx.fee', 2),
-                    detail: `${transaction.feeFormatted.value} ${this.$i18n.t('common.eth')}`,
-                    tooltip: transaction.feeFormatted.tooltipText ? `${transaction.feeFormatted.tooltipText} ${this.$i18n.t('common.eth')}` : undefined
+                    detail: this.transaction.value
```
Comment (olgakup) [inline newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue:171]: formatVariableUnitEthValue()

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue [hunk 3]
```diff
@@ -173,121 +143,103 @@ export default class PageDetailsTxs extends Vue {
...
+                // {
+                //     title: this.$i18n.tc('tx.fee', 2),
+                //     detail: `${this.transaction.feeFormatted.value} ${this.$i18n.t('common.eth')}`,
+                //     // tooltip: this.transaction.feeFormatted.tooltipText ? `${this.transaction.feeFormatted.tooltipText} ${this.$i18n.t('common.eth')}` : undefined
+                // },
                 {
                     title: this.$i18n.t('gas.limit'),
-                    detail: transaction.gasFormatted.value,
-                    tooltip: transaction.gasFormatted.tooltipText ? `${transaction.gasFormatted.tooltipText}` : undefined
+                    detail: this.transaction.gas
```
Comment (olgakup) [inline newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue:183]: fomatNumber()

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue [hunk 4]
```diff
@@ -173,121 +143,103 @@ export default class PageDetailsTxs extends Vue {
...
+                    // tooltip: this.transaction.gasFormatted.tooltipText ? `${this.transaction.gasFormatted.tooltipText}` : undefined
                 },
                 {
                     title: this.$i18n.t('gas.used'),
-                    detail: receipt ? receipt.gasUsedFormatted.value : '0', // genesis block txs can have no receipt
-                    tooltip: receipt && receipt.gasUsedFormatted.tooltipText ? `${receipt.gasUsedFormatted.tooltipText}` : undefined
+                    detail: this.transaction.gasUsed // TODO genesis block txs can have no receipt
```
Comment (olgakup) [inline newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue:188]: formatNumber()

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue [hunk 5]
```diff
@@ -173,121 +143,103 @@ export default class PageDetailsTxs extends Vue {
...
+                    // tooltip: receipt && receipt.gasUsedFormatted.tooltipText ? `${receipt.gasUsedFormatted.tooltipText}` : undefined
                 },
                 {
                     title: this.$i18n.t('gas.price'),
-                    detail: `${transaction.gasPriceFormatted.value} ${this.$i18n.t(`common.${transaction.gasPriceFormatted.unit}`)}`,
-                    tooltip: transaction.gasPriceFormatted.tooltipText ? `${transaction.gasPriceFormatted.tooltipText}` : undefined
+                    detail: this.transaction.gasPrice
```
Comment (olgakup) [inline newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue:193]: formatVariableUnitEthValue()

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue (hunk 1 above)
Comment (gesseekur) [reply newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue]: updated!

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue (hunk 2 above)
Comment (gesseekur) [reply newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue]: updated

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue (hunk 3 above)
Comment (gesseekur) [reply newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue]: updated

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue (hunk 4 above)
Comment (gesseekur) [reply newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue]: updated

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue (hunk 5 above)
Comment (gesseekur) [reply newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue]: updated

File: newclient/src/modules/blocks/handlers/BlockDetails/BlockDetails.vue [hunk 6]
```diff
@@ -155,6 +155,7 @@ export default class BlockDetails extends Mixins(NumberFormatMixin, NewBlockSubs
                 }
             ]
         } else {
+            console.log('block', this.uncleRewards)
```
Comment (olgakup) [inline newclient/src/modules/blocks/handlers/BlockDetails/BlockDetails.vue:158]: delete console log please

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue [hunk 7]
```diff
@@ -143,10 +146,11 @@ export default class TxDetails extends Vue {
             ]
         } else {
             // const receipt = transaction.receipt!
+            console.log('adf', this.txFee)
```
Comment (olgakup) [inline newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue:149]: delete console log please

File: newclient/src/modules/blocks/handlers/BlockDetails/BlockDetails.vue (hunk 6 above)
Comment (gesseekur) [reply newclient/src/modules/blocks/handlers/BlockDetails/BlockDetails.vue]: removed!

File: newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue (hunk 7 above)
Comment (gesseekur) [reply newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue]: removed!

**Pull request merged**